docs/apache-airflow/howto/set-up-database.rst

```diff
@@ -300,6 +300,14 @@ We recommend using the ``mysqlclient`` driver and specifying it in your SqlAlche
 
 We also support the ``mysql-connector-python`` driver, which lets you connect through SSL
 without any cert options provided.
+Unfortunately, this package depends on outdated version of ``protobuf<4.0``  which is no longer supported.
+If you wish to use ``mysql-connector-python`` driver, please install it with pip.
+
+.. code-block:: text
+
+   $ pip install mysql-connector-python
```

Comment:
Why? We enabled it as extra so the install should be with the extra

Reply:
Thanks for noticing. Fixed.